lib/libzrepl/data_conn.c

```diff
@@ -740,6 +757,9 @@ uzfs_zvol_rebuild_dw_replica(void *arg)
 
```

Comment:
Not related to this PR. Do we need to have quiesce_wait for all threads in case of mesh rebuild as per the design?